complaint_search/defaults.py

```diff
@@ -9,4 +9,26 @@
 }
 
 # -*- coding: utf-8 -*-
-DELIMITER = u'\u2022'
\ No newline at end of file
+DELIMITER = u'\u2022'
+
+SOURCE_FIELDS = (
+    "sub_product",
```

Comment:
Can this list be sorted? Or is order important?

Reply:
order is not important, I will sort them